### PR TITLE
Arr Execise

### PR DESCRIPTION
**Pull request by boone824**
Add Arr Exercise Assignment

### CODE REVIEW

**Comment by boone824** [comment]
Added Exercise 2

**Comment by boone824** [comment]
Add Exercise 3

File: lesson3/exercise-two.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let breakfastqt =['Bacon', 'and','eggs','are', 'delicious!'];
+
+const breakfastqtbrand = breakfastqt.join(' ');
+console.log(breakfastqtbrand);
+console.log('eggs')
```
Comment (jimmynono) [inline lesson3/exercise-two.js:5]: Yes, this is technically accurate, but I'm hoping you can use the array[] syntax to target the 3rd word

**Comment by boone824** [comment]
Added Assignment 1 Function

**Pull request closed without merging**